docs/index.txt

```diff
@@ -131,48 +129,23 @@ Properties can be specified when connecting to Crate using the JDBC driver::
 
     Properties properties = new Properties();
     properties.put(<key>, <value>);
-    Connection conn = DriverManager.getConnection("crate://localhost:4300", properties);
+    Connection conn = DriverManager.getConnection("crate://localhost:5432/", properties);
 
 In addition connection properties can be passed via the JDBC URL::
 
-    Connection conn = DriverManager.getConnection("crate://localhost:4300?property1=value1&property2=value2");
+    Connection conn = DriverManager.getConnection("crate://localhost:5432/?property1=value1&property2=value2");
 
 Crate JDBC driver supports following properties:
 
 :strict:
     **Default**: ``false``.
 
-    Setting the strict property to ``true`` will
-    enforce the driver to be compliant with the JDBC specification.
-    Therefore, calling any operations that are not supported by the
-    driver will result in an exception. For instance, disabling the
-    auto-commit mode or setting a savepoint will raise an unsupported
-    operation exception.
-    By default the ``strict`` property is optimistically set to ``false``
-    which will make the Crate JDBC driver compatible with most 3rd party
-    applications that usually require transactional databases.
-
-:showsubcolumns:
-    **Default**: ``false``.
-
-    .. warning:: This property is experimental and may be subject to change in
-                 future releases.
-
-    If this property value is set to ``true`` the method ``getColumns()``
-    will also list all subcolumns for columns of type ``OBJECT``.
-    The output is then equivalent to::
-
-        SELECT column_name
-        FROM information_schema.columns
-        WHERE schema_name = ? AND table_name = ?
-
-    By setting the property to ``false`` only the root column (the object
-    field name) itself is returned.
-
-    .. note:: 3rd party tool ususally quote the identifiers obtained by the
-              ``getColumns()`` method. Quoting needs to be disabled when this
-              property is set to ``true``. This however limits the naming of
-              columns, because they must only contain lower case ASCII letters.
+    If you set this property to ``true`` the driver will be compliant with the JDBC
+    specification. It will then raise an unsupported operation exception if you
+    call an operation which is not supported, like auto-commit or setting a
```

Comment:
Would use 3rd person:
It will raise an Exception if an unsupported operation is called.